lib/features/children/generosity_challenge/utils/generosity_challenge_content_helper.dart

```diff
@@ -56,10 +57,18 @@ class GenerosityChallengeContentHelper {
     ),
     Task.card(
       image: 'assets/images/generosity_challenge_day_7.svg',
-      title: 'What a look of surprise shone on Mastro Cherry’s face!',
+      title: 'Giving together',
+      buttonText: 'Find a charity',
       description:
-          'His funny face became still funnier. He turned frightened eyes about the room to find out where that wee, little voice had come from and he saw no one! He looked under the bench—no one! He peeped inside the closet—no one! He searched among the shavings—no one!',
+          'Using your Family Values see what charities align with what you care about. Give to one of them.',
+      redirect: Pages.displayValues.path,
       onTap: () {},
+      partnerCard: Task.card(
+        image: 'assets/images/generosity_challenge_day_7.svg',
+        title: 'Giving together',
+        description: 'You gave 15 dollars to the American Red Cross. Awesome!',
```

Comment:
Not sure if it's related to this task or a different task for in the future but is this description supposed to be hardcoded? "you gave 15...to red cross"

Reply:
It's a placeholder, can be empty, in this flow the partner card will get a dynamic description that overwrites this one on the GenerosityDailyCard, happens in day 2 also